docs/book/src/developer/architecture/controllers/cluster.md

```diff
@@ -5,6 +5,7 @@
 The Cluster controller's main responsibilities are:
 
 * Setting an OwnerReference on the infrastructure object referenced in `Cluster.Spec.InfrastructureRef`.
+* Setting an OwnerReference on the control plane object referenced in `Cluster.Spec.ControlPlaneRef`.
```

Comment:
```suggestion
* Setting an OwnerReference on the controlPlane object referenced in `Cluster.Spec.ControlPlaneRef`.
```
nits - these names are better aligning with the way the structs are named (+/- capitaliztion) to prevent any confusion.

Reply:
I would probably use:
```suggestion
* Setting an OwnerReference on the ControlPlane object referenced in `Cluster.spec.controlPlaneRef`.
```
(changed the path, not sure if we have any consistency in the docs regarding "control plane object" vs "ControlPlane object")

Reply:
It seems It's referred across this file and also in control-plane.md as "control plane" so I'd rather keep it and consider renaming everywhere in a different PR.

Reply:
That's fine by me - control plane is easy to understand written as plain English. For InfrastructureCluster and others though it tends to be clearer to make the fact it's a type very explicit.

Reply:
@enxebre WDYT about correcting the JSON path? `Cluster.Spec.ControlPlaneRef` => `Cluster.spec.controlPlaneRef`
(I don't insist on it)

Reply:
I had missed it, done! updated the existing one a the one this is introducing.